tflite2onnx/op/padding.py

```diff
@@ -15,6 +15,7 @@
 class Padding(Operator):
     TypeMapping = {
         tflite.BuiltinOperator.PAD: 'Pad',
+        tflite.BuiltinOperator.MIRROR_PAD: 'MirrorPad',
```

Comment:
Should this be `Pad`? The value of `TypeMapping` is the op type of ONNX.

Reply:
We can also update the [`type`](https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R29) to use `TypeMapping`.

Reply:
https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R18

https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R29